src/sass/_base.scss

```diff
@@ -0,0 +1,146 @@
+html {
+    box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+body {
+    color: $color-text-body;
+    font-family: "Commissioner", sans-serif;
+    font-weight: 400;
+    margin: 0;
+    height: 100vh;
+}
+
+.sidenav {
+  background-color: $color-grey;
+  position: fixed;
+  height: 100%;
+  width: 432px;
```

Comment:
I know you were only starting to get going with responsive styles, but 2 things to point out:
1. For most modern web sites, you want to start with mobile styles and then add on your media queries.
2. You may benefit here from using more relative measures (like a % or viewport width) for the width and giving it a `max-width` to prevent it from ever getting too large.